### ISSUE TITLE
[BUG] Convergence issues with pytorch dataloader on criteo dlrm

### ISSUE THREAD

**Issue by benfred** [NVIDIA/NVTabular#157]

**Comment by jperez999** [comment]
Ran the synchronous version hit the nan error at the same time as the asynchronous version on 50% of runs. So far, that is comparable to the asynchronous version.

**Comment by jperez999** [comment]
Reduced learning rate by 1 to 27, still hit nan problem

**Comment by jperez999** [comment]
Reduced max learning rate to 20 and have not encountered a failure yet.

**Comment by benfred** [comment]
Closing - feel free to reopen if this is still happenning
